### PR TITLE
configuration variables moved to appsettings

### PR DESCRIPTION
**Pull request by Dragonfrut**
appsettings added to ignore

### CODE REVIEW

**Comment by Dragonfrut** [comment]
The naming convention was also fixed in a few areas

**Comment by Dragonfrut** [comment]
Configuration changes appear to be in working order. there is an issue in DataBaseService line 16: System.ArgumentNullException: 'Value cannot be null. (Parameter 'name')

File: WildShape-Sheets-API/Services/DataBaseService.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Options;
```
Comment (smashingmouse) [inline WildShape-Sheets-API/Services/DataBaseService.cs:1]: Awesome! Database service abstracted!

File: WildShape-Sheets-API/Services/PlayerCharacterService.cs [hunk 2]
```diff
@@ -1,42 +1,39 @@
...
-        private readonly IMongoCollection<PlayerCharacter> playerChars;
+        private readonly DataBaseService _dataBaseService;
+        private readonly IConfiguration _configuration;
 
-        public PlayerCharacterService(IOptions<WildshapeSheetsDBSettings> wildshapeSheetsDBSettings, IConfiguration configuration)
+        public PlayerCharacterService(IConfiguration configuration, DataBaseService dataBaseService)
         {
-            var mongoClient = new MongoClient(
-                wildshapeSheetsDBSettings.Value.ConnectionString);
+            _dataBaseService = dataBaseService;
```
Comment (smashingmouse) [inline WildShape-Sheets-API/Services/PlayerCharacterService.cs:15]: Woot! Model services have no knowledge or care about the underlying database implementation. Mongo or Postgresql or Chocolate Chip, it is abstracted away and could be changed to anything in one place (in theory).

File: WildShape-Sheets-API/appsettings.json [hunk 3]
```diff
@@ -7,7 +7,14 @@
     "IsSSL": true
   },
 
-  "JwtKey": "itzaquey", 
+  "APIKeys": {
```
Comment (smashingmouse) [inline WildShape-Sheets-API/appsettings.json:10]: Collecting config setting in one place, love it!

**Review by smashingmouse — APPROVED** [review]
Not that you need my approval, but you have it :) This shows excellent understanding of some core principals. Nicely done!

**Pull request merged**